code/ATMOSPHERICS/hvac/spaceheater.dm

```diff
@@ -335,23 +335,30 @@
 	light_range_on = 6
 	nocell = 2
 	density = 0
+	pixel_y = 30
```

Comment:
@TermedClepe it uses this to appear on the wall but not be on it, like every other item that's on a wall on one side and not the other.